style.css

```diff
@@ -0,0 +1,92 @@
+*,
+*::after,
```

Comment:
no need to write pseudo element after and before only  * is ok

Reply:
if i remove them it creates a gap between the text and buy now button please check